src/include/storage/write_ahead_log/log_record.h

```diff
@@ -150,15 +159,18 @@ class RedoRecord {
    * @param head
    * @param size
    * @param txn_begin
-   * @param table
+   * @param db_oid database oid of this redo record
+   * @param table_oid table oid of this redo record
    * @param tuple_slot
    * @return
    */
   static LogRecord *PartialInitialize(byte *const head, const uint32_t size, const transaction::timestamp_t txn_begin,
-                                      DataTable *const table, TupleSlot tuple_slot) {
+                                      const catalog::db_oid_t db_oid, const catalog::table_oid_t table_oid,
+                                      const TupleSlot tuple_slot) {
```

Comment:
Based off the TODO above, we should probably remove this function or at a minimum make it private with the logging test as a friend to prevent this from getting used in @GustavoAngulo replication code.

Reply:
#370 removes it so I'm not gonna worry about it in this PR.